src/app/login/login-form.component.html

```diff
@@ -18,7 +18,9 @@
     </mat-error>
   </mat-form-field>
   <button mat-button [planetSubmit]="userForm.valid">{{ createMode ? 'Become a member' : 'SIGN-IN' }}</button>
-  <a class = "addUser" [routerLink]="createMode ? ['/login'] : ['newmember']">
-    {{ createMode ? 'Already have an account?' : 'Become a member' }}
-  </a>
+  <button mat-button>
+    <a class = "addUser" [routerLink]="createMode ? ['/login'] : ['newmember']">
+      {{ createMode ? 'Already have an account?' : 'Become a member' }}
+    </a>
+</button>

```

Comment:
Please be careful to make sure you have the correct indentation in the HTML files.  This will not get picked up by the linter